tests/tests/namespace/run-prepare-functions.spec.js

```diff
@@ -7,26 +7,56 @@ const API_RULE_AND_FUNCTION_NAME = 'in-cluster-eventing-publisher';
 
 context('Prepare funtions for testing', () => {
   before(() => {
+    cy.setBusolaFeature('EXTENSIBILITY', true);
+    cy.mockExtension(
+      'FUNCTIONS',
+      'examples/resources/serverless/functions.yaml',
+    );
+
     cy.loginAndSelectCluster();
     cy.goToNamespaceDetails();
   });
 
   it('Create a simple Function', () => {
+    cy.createSimpleFunction(FUNCTION_NAME);
+  });
+
+  it('Create a receiver Function', () => {
+    cy.createFunction(
+      FUNCTION_RECEIVER_NAME,
+      'fixtures/in-cluster-eventing-receiver.js',
+      'fixtures/in-cluster-eventing-receiver-dependencies.json',
+    );
+  });
+
+  it('Create a publisher Function', () => {
+    cy.createFunction(
+      API_RULE_AND_FUNCTION_NAME,
+      'fixtures/in-cluster-eventing-publisher.js',
+      'fixtures/in-cluster-eventing-publisher-dependencies.json',
+    );
+  });
+
+  // edit test case is the last one because of the following error:
+  // "the object has been modified; please apply your changes to the latest
+  // version abnd try again"
+  // we need to wait until the Function isn't modified in the meantime
```

Comment:
I'm not sure what you mean here. Maybe this:  in the meantime we need to wait until the Function IS modified

Reply:
I will change it to: "we must wait for the function to remain unchanged". What do you think?